_build/partials/interactive-map.hbs

```diff
@@ -0,0 +1,57 @@
+<div id="leaflet-map"></div>
+<script>
+    async function loadData(){
+        const pointsResponse = await fetch("{{geojsonPointsUrl}}");
+        const geojsonPoints = await pointsResponse.json();
+
+        const polygonsResponse = await fetch("{{geojsonPolygonsUrl}}");
+        const geojsonPolygons = await polygonsResponse.json();
+
+        return {geojsonPoints, geojsonPolygons };
+    }
+
+    function generatePopupText(feature){
+        const functionString = () => navigator.clipboard.writeText(JSON.stringify(feature.geometry))
+
+        return "<h4>title</h4>" +
+            "<div>" +
+            "Time range:" + feature.properties.from + " to " + feature.properties.to + "<br>" +
+            "Area size: " + feature.properties["area / sqkmm"] + " km<sup>2</sup><br>" +
```

Comment:
Can we add some more padding between button and text as in the designs? It currently looks quite cramped.

![grafik](https://github.com/sentinel-hub/public-collections/assets/60705209/e270ffb6-5e51-4b4d-a993-6e130385980c)

Reply:
thanks, done